http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/DiskV2ServiceInterpSpec.scala

```diff
@@ -192,9 +192,36 @@ class DiskV2ServiceInterpSpec extends AnyFlatSpec with LeonardoTestSuite with Te
           )
         )
       )
-      err <- diskV2Service.deleteDisk(userInfo, workspaceId, disk.id).attempt
+      err <- diskV2Service.deleteDisk(userInfo, disk.id).attempt
     } yield err shouldBe Left(DiskCannotBeDeletedAttachedException(disk.id, workspaceId, ctx.traceId))
 
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
+
+  it should "fail to delete a disk if it has no workspaceId" in isolatedDbTest {
```

Comment:
Is it possible to create a disk without a workspaceId?

Reply:
Adding in this logic concerns me, it seems to be just another layer of obscure functionality that restricts user control - if this scenario happens, manual intervention is required.

Reply:
with an Azure disk no, but with a GCP runtime yes. The workspaceId column in the PERSISTENT_DISK table is an option, so in order to get the workspaceId I used that fromOption logic. I don't think this exception should happen very often/at all, but thought it might be useful when we switch over to using the v2 routes for GCP as well.

Reply:
yup yeah that's why its needed in the backend call, every call to WSM for azure disks, runtimes etc requires a workspaceId